packages/plugin-board/test/integration/spec/board.js

```diff
@@ -0,0 +1,414 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import '../..';
+
+import {assert} from '@ciscospark/test-helper-chai';
+import CiscoSpark from '@ciscospark/spark-core';
+import testUsers from '@ciscospark/test-helper-test-users';
+import {flaky} from '@ciscospark/test-helper-mocha';
+import fh from '@ciscospark/test-helper-file';
+import {find, map} from 'lodash';
+import uuid from 'uuid';
+
+function generateTonsOfContents(numOfContents) {
+  const contents = [];
+
+  for (let i = 0; i < numOfContents; i++) {
+    contents.push({
+      type: `curve`,
+      payload: JSON.stringify({id: i, type: `curve`})
+    });
+  }
+  return contents;
+}
+
+function boardChannelToMercuryBinding(channelId) {
+  // make channelId mercury compatible replace `-` with `.` and `_` with `#`
+  return channelId.replace(/-/g, `.`).replace(/_/g, `#`);
+}
+
+describe(`plugin-board`, function() {
+  this.timeout(30000);
+
+  let board, conversation, fixture, participants;
+  const mercuryBindingsPrefix = `board.`;
+
+  // create users
```

Comment:
@ianwremmel, I like the idea of using multiple before blocks.  I am going to do that with the  fixtures for sure, but each of the blocks depend on the one before it.  In my testing, it seems that issuing several before blocks will execute all the blocks in parallel.  This is not the desired behavior.

I refactored this file to make the tests less confusing.  We had `ensureConversation` and `ensureBoard` in the before block of each describe block.  This was nice, but I realized that we just needed one setup for all the tests, so I consolidated into one before block in the parent describe.  This makes the tests more readable, I believe.  I could go back to the other way if you disagree.